tests/Common/BranchComponentTest.php

```diff
@@ -261,6 +261,23 @@ function ($branch) use ($branchName) {
         $this->assertEquals('{"id":"10","stuff":"true"}', $updatedRow['configuration'][0]);
         $this->assertEquals(1, $updatedRow['version']);
 
+        $branchComponents->updateConfigurationRow(
```

Comment:
Proč je to tady? Není to otestované na ř. 247-262?

Reply:
Jo jasne diky, pomylil som si to s `main-1-row-1` odstranil som to.